src/main/java/carracinggame/controller/CarRacing.java

```diff
@@ -0,0 +1,51 @@
+package carracinggame.controller;
+
+import carracinggame.domain.Car;
+import carracinggame.domain.Cars;
+import carracinggame.domain.Times;
+import carracinggame.domain.Winner;
+import carracinggame.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarRacing {
+
+    int carsCount;
+
+    Cars cars = new Cars();
+
+    public void start(String[] carNames, int times) {
+        cars.makeCars(carNames);
+        carsCount = carNames.length;
+        OutputView.printResultMessage();
+        playRacingGame(times);
+    }
+
+    private void playRacingGame(int times) {
+        while (!Times.isEnd(times)) {
+            cars.run();
+            turnResult();
+            times--;
+        }
+    }
+
+    private void turnResult() {
+        for (int i = 0; i < carsCount; i++) {
+            Car car = cars.getCar(i);
+            OutputView.printResult(car.getName(), car.getPosition());
+        }
+        OutputView.println();
+    }
```

Comment:
자동차 객체들이 있는 리스트가 Cars 클래스에 있어서 controller에서 정보를 가져오려고 이렇게 구성했는데 컨트롤러에서 cars.getCar(i)부분을 이런식으로 car객체를 가져오는게 맞나요?

Reply:
```suggestion
    private void turnResult() {
        for(Car car: cars.getCars()) {
            OutputView.printResult(car.getName(), car.getPosition());
        }
        OutputView.println();
    }
```

이런식으로 for each를 활용할 수 있을 것 같아요 😊

https://wikidocs.net/264

Reply:
car.getCars()라는 메서드는 어떤건 나타낸 건가요??

Reply:
임의로 cars에서 cars를 가져오는 게터를 만들었다는 설명을 콜을 통해 했습니다~!